includes/definitions/discovery/aos7.yaml

```diff
@@ -121,7 +121,7 @@ modules:
                 -
                     oid: chasControlModuleTable
                     value: chasControlSynchronizationStatus
-                    num_oid: '.1.3.6.1.4.1.6486.801.1.1.1.3.1.1.1.1.4.{{ $index }}'
+                    num_oid: '.1.3.6.1.4.1.6486.801.1.1.1.3.1.1.1.1.5.{{ $index }}'
```

Comment:
According to the mib, it seems to be still the wrong OID : 
1.3.6.1.4.1.6486.800.1.1.1.3.1.1.1.1.6 | chasControlSynchronizationStatus

Reply:
Fixed. Add some other changes. Still the same. 

> OS aos7: Discovered ports data does not match that found in tests/data/aos7.json

This is something new to me. I tried to fix only some sensors. Can be that something was actually broken before?